test/jit/test_complex.py

```diff
@@ -55,3 +58,75 @@ def fn(a: int, b: torch.Tensor, dim: int):
         t2 = torch.tensor([0.4, 1.4j, 2.35])
 
         self.checkScript(fn, (t1, t2, 2))
+
+    def test_complex_math_ops(self):
+        inf = float("inf")
+        NaN = float("nan")
+        vals = ([inf, NaN, 0.0, 1.0, 2.2, -1.0, -0.0, -2.2, -inf, 1, 0, 2]
+                + [10.0 ** i for i in range(5)] + [-(10.0 ** i) for i in range(5)])
+        complex_vals = tuple(complex(x, y) for x, y in product(vals, vals))
+
+        def checkMath(func_name, num_args, ret_type="complex", debug=False, vals=None, args_type=None):
+            funcs_template = dedent('''
+            def func(a):
+                # type: {args_type} -> {ret_type}
```

Comment:
```suggestion
            def func(a: complex) -> {ret_type}:
```
Python 2 is no longer supported.